Northstar.CustomServers/mod/scripts/vscripts/_powerup.gnut

```diff
@@ -21,9 +34,16 @@ void function AddPowerupSpawn( entity spawnpoint )
 
 void function RespawnPowerups()
 {
+	string map = GetMapName()
```

Comment:
Can't this be implement in the specific map scripts? We try to keep these patches there as much as possible, example: https://github.com/R2Northstar/NorthstarMods/blob/main/Northstar.CustomServers/mod/scripts/vscripts/mp/levels/mp_grave.nut